__test__/event.spec.js

```diff
@@ -69,5 +89,96 @@ describe("testing event api routes", () => {
                 assert.deepEqual(res.body, event1)
             })
     });
+    let event2;
+    it("Create course events", async () => {
```

Comment:
Multiple asserts in the same test is bad for debugging (if this test fails, which assertion is the culprit?).

Reply:
> Multiple asserts in the same test is bad for debugging (if this test fails, which assertion is the culprit?).
The console specifies which line the assert fails. I'm also pretty sure it will end that specific test at the failed assert line so it is pretty clear which assert failed.

Reply:
Is that also the case when you run all the tests together? Or is this only applicable when running one test at a time?

Reply:
All of the tests

Reply:
Alright I just tested it and it looks like the test stops at the line the assert fails and does not proceed further. (This makes sense as the rest of the test might require this.
![image](https://user-images.githubusercontent.com/59633304/207209042-56fce046-4272-4def-8a23-f80796c0a504.png)

Reply:
Context of the image above
![image](https://user-images.githubusercontent.com/59633304/207209092-081c2e25-8a53-457e-992e-83414fe816ce.png)
There should be 2 failed asserts, but console will only show the first one as it stops testing.

Reply:
That's a little odd. When I was checking test dependence for the previous sprint, my tests ran until the end even when one of the tests failed. I'll need to recheck this on my end.